x-pack/plugins/enterprise_search/public/applications/shared/search_labs_banner/search_labs_banner.tsx

```diff
@@ -72,7 +73,7 @@ export const SearchLabsBanner: React.FC = () => {
           <span>
             <EuiButton
               href="https://www.elastic.co/search-labs/tutorials"
-              color="success"
+              color="accentSecondary"
```

Comment:
@andreadelrio If the button sits inside a panel, shouldn't it follow that panels color? So does this mean we can no longer have an `accentSecondary` panel?

Reply:
As agreed in Slack, will keep this as success panel with a success button.